feature/interface/singleton/otg_tests/singleton_test/singleton_test.go

```diff
@@ -26,10 +26,12 @@ import (
 	"github.com/openconfig/featureprofiles/internal/fptest"
 	"github.com/openconfig/featureprofiles/internal/otgutils"
 	"github.com/openconfig/ondatra"
+	"github.com/openconfig/ygnmi/ygnmi"
 	"github.com/openconfig/ygot/ygot"
 
```

Comment:
Any idea why goimports leaves empty lines in places like this?

Reply:
empty lines between imports are generally used to group imports, i don't think a particular grouping is enforced by goimports, except that imports are sorted in a group